index.d.ts

```diff
@@ -0,0 +1,5 @@
+import {GraphQLSchema} from "graphql";
+
+export function constraintDirective () : (schema: GraphQLSchema) => GraphQLSchema;
+
+export const constraintDirectiveTypeDefs: String
```

Comment:
`String` should be `string`

Reply:
@ltphen will merge this in once the comments have been addressed. Thanks!

Reply:
Hey @confuser, The changes has been taken into account